### PR TITLE
Bump version to v0.1.14

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

**Review by zanieb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+- \[`pygrep_hooks`\] Add fix for `deprecated-log-warn` (`PGH002`) ([#9519](https://github.com/astral-sh/ruff/pull/9519))
+- \[`pylint`\] Implement `import-private-name` (`C2701`) ([#5920](https://github.com/astral-sh/ruff/pull/5920))
+- \[`refurb`\] Implement `regex-flag-alias` with fix (`FURB167`) ([#9516](https://github.com/astral-sh/ruff/pull/9516))
+- \[`ruff`\] Add rule and fix to sort contents of `__all__` (`RUF022`) ([#9474](https://github.com/astral-sh/ruff/pull/9474))
+- \[`tryceratops`\] Add fix for `error-instead-of-exception` (`TRY400`) ([#9520](https://github.com/astral-sh/ruff/pull/9520))
+
+### Rule changes
+
+- \[`flake8-pyi`\] Fix `PYI047` false negatives on PEP-695 type aliases ([#9566](https://github.com/astral-sh/ruff/pull/9566))
+- \[`flake8-pyi`\] Fix `PYI049` false negatives on call-based `TypedDict` ([#9567](https://github.com/astral-sh/ruff/pull/9567))
```
Comment (AlexWaygood) [inline CHANGELOG.md:18]:
```suggestion
- \[`flake8-pyi`\] Fix `PYI049` false negatives on call-based `TypedDict`s ([#9567](https://github.com/astral-sh/ruff/pull/9567))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+- \[`pylint`\] Exclude `self` and `cls` when counting method arguments ([#9563](https://github.com/astral-sh/ruff/pull/9563))
```
Comment (AlexWaygood) [inline CHANGELOG.md:19]:
```suggestion
- \[`pylint`\] Exclude `self` and `cls` when counting method arguments (`PLR0917`) ([#9563](https://github.com/astral-sh/ruff/pull/9563))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+- Add `--extension` support to the formatter ([#9483](https://github.com/astral-sh/ruff/pull/9483))
+
+### Configuration
+
+- Ignore preview status for fixable and unfixable selectors ([#9538](https://github.com/astral-sh/ruff/pull/9538))
+- \[`pycodestyle`\] Use the configured tab size when expanding indents ([#9506](https://github.com/astral-sh/ruff/pull/9506))
+
+### Bug fixes
+
+- Add an explicit fast path for whitespace to `is_identifier_continuation` ([#9532](https://github.com/astral-sh/ruff/pull/9532))
```
Comment (AlexWaygood) [inline CHANGELOG.md:33]: is this a bugfix or a performance feature?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+- Recursively visit deferred AST nodes ([#9541](https://github.com/astral-sh/ruff/pull/9541))
+- Visit deferred lambdas before type definitions ([#9540](https://github.com/astral-sh/ruff/pull/9540))
+- \[`flake8-simplify`\] Avoid some more `enumerate-for-loop` false positives (`SIM113`) ([#9515](https://github.com/astral-sh/ruff/pull/9515))
+- \[`pandas-vet`\] Limit inplace diagnostics to methods that accept inplace ([#9495](https://github.com/astral-sh/ruff/pull/9495))
+- \[`pylint`\] Add the `__prepare__` method to the list of recognized dunder method ([#9529](https://github.com/astral-sh/ruff/pull/9529))
+- \[`pylint`\] Ignore unnecessary dunder calls within dunder definitions ([#9496](https://github.com/astral-sh/ruff/pull/9496))
+- \[`refurb`\] Avoid bailing when `reimplemented-operator` is called on function ([#9556](https://github.com/astral-sh/ruff/pull/9556))
```
Comment (AlexWaygood) [inline CHANGELOG.md:40]:
```suggestion
- \[`refurb`\] Avoid bailing when `reimplemented-operator` is called on function (`FURB118`) ([#9556](https://github.com/astral-sh/ruff/pull/9556))
```

**Review by AlexWaygood — COMMENTED** [review]
LG, a few small suggestions:

**Review by AlexWaygood — APPROVED** [review]

**Comment by charliermarsh** [comment]
Thx!

**Pull request merged**